var/spack/repos/builtin/packages/pika/package.py

```diff
@@ -96,7 +96,7 @@ class Pika(CMakePackage, CudaPackage, ROCmPackage):
     conflicts("cxxstd=23", when="^cmake@:3.20.2")
     # CUDA version <= 11 does not support C++20 and newer
```

Comment:
Maybe we can update this to the more accurate:
```suggestion
    # nvcc version <= 11 does not support C++20 and newer
```

Reply:
yes tks for noticing!